libraries/Microsoft.Bot.Builder.AI.QnA/Dialogs/QnAMakerDialog.cs

```diff
@@ -636,6 +636,13 @@ private async Task<DialogTurnResult> CheckForMultiTurnPromptAsync(WaterfallStepC
             return await stepContext.NextAsync(stepContext.Result, cancellationToken).ConfigureAwait(false);
         }
 
+        /// <summary>
+        /// Gets the options the dialog will use to display query results to the user.
+        /// </summary>
+        /// <param name="stepContext">The <see cref="WaterfallStepContext"/> for the current turn of conversation.</param>
+        /// <param name="cancellationToken">The <see cref="CancellationToken"/> for the current turn of conversation.</param>
+        /// <returns>A <see cref="Task"/> representing the asynchronous operation.</returns>
+        /// <remarks>If the task is successful, the result contains the response options to use.</remarks>
         private async Task<DialogTurnResult> DisplayQnAResultAsync(WaterfallStepContext stepContext, CancellationToken cancellationToken)
```

Comment:
can you make these methods protected now?

Reply:
Will be moved to next PR